### ISSUE TITLE
Low Brightness issue 

### ISSUE THREAD

**Issue by namanagrawal94** [phhusson/treble_experimentations#2365]
### Smartphone (please complete the following information):
 - Device: Micromax In 1
 - Build Flavor: system-squeak-arm64-ab-gapps.img.xz
 - Version : V414

### Additional context
I have set the maximum brightness on my phone but still the brightness feels quite low. I have made the changes from phh settings i e, Force alternative backlight scale also but it simply does not work. I would request you to consider this issue with the next build.

**Comment by phhusson** [comment]
Does the brightness slider work properly except for that?
Give the result of `dumpsys -l` and of `lshal`

Le mer. 15 juin 2022 à 10:03, namanagrawal94 ***@***.***> a
écrit :

**Comment by namanagrawal94** [comment]
Yes the brightness slider works fine. I am attaching the dumpsys file and lshal for your reference.
[dumpsys.txt](https://github.com/phhusson/treble_experimentations/files/8907290/dumpsys.txt)

**Comment by namanagrawal94** [comment]
[lshal.txt](https://github.com/phhusson/treble_experimentations/files/8907315/lshal.txt)
[dumpsys.txt](https://github.com/phhusson/treble_experimentations/files/8907318/dumpsys.txt)

Yes the brightness slider works fine. I am attaching the dumpsys file and lshal for your reference.

**Comment by phhusson** [comment]
Slider works fine both if you select "use alternate backlight" and if you don't?

**Comment by namanagrawal94** [comment]
If i dont select force alternate backlight, the slider works fine but still the brightness is very very low. If I select force alternate backlight, the brightness gets even worse.

**Comment by namanagrawal94** [comment]
Yes, If I enable or disable, brightness is still very low. Only thing is if I enable, the brightness gets even lower.

**Comment by phhusson** [comment]
In both cases, the slider works fine? when you go from lowest brightness to highest brightness on the slider, the brightness keeps increasing, and never decrease?

**Comment by namanagrawal94** [comment]
Yes slider works fine. If i increase the brightness, brightness increases and If i decrease it, it decreases as well. Only thing is max brightness is very very low in both the cases. If i keep my phone at 50% brightness, it feels like I am using at 20% as on stock rom

**Comment by namanagrawal94** [comment]
I was on corvus gsi before, it used to work fine before. I request you to fix this on aosp gsi as well.

**Comment by phhusson** [comment]
> I request you to fix this on aosp gsi as well.

Did you think I was discussing with you in order NOT TO fix your issue?

> I was on corvus gsi before, it used to work fine before

Yeah well corvus developers didn't send me any fix, so I'm not sure what
I'm supposed to do with this information.

**Comment by namanagrawal94** [comment]
Sorry sir, please accept my apology.

**Comment by phhusson** [comment]
Try setting persist.sys.qcom-brightness to 8192 and make a video showing
the brightness behavior of the whole brightness slider

Le mer. 15 juin 2022 à 13:29, namanagrawal94 ***@***.***> a
écrit :

**Comment by namanagrawal94** [comment]
If i dont select force alternate backlight, the slider works fine but still
the brightness is very very low. If I select force alternate backlight, the
brightness gets even worse.
